src/styles.tsx

```diff
@@ -0,0 +1,101 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyles = createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+  @import url('https://fonts.googleapis.com/css?family=Libre+Baskerville&display=swap');
+  @import url('https://fonts.googleapis.com/css?family=Rubik&display=swap');
```

Comment:
Creating a PR to remove this because we realized it didn't work, and because styled-components gives a warning about not doing this.